campaigns.json

```diff
@@ -2759,16 +2786,25 @@
     "go": false,
     "lumisize": -1,
     "maxcopies": 1,
-    "SecondaryLocation": [
-       "T1_DE_KIT_Disk",
-       "T1_US_FNAL_Disk"
-    ],
     "SiteBlacklist": [
        "T2_CH_CERN",
        "T2_CH_CERN_HLT"
      ],
     "secondaries": {
-       "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL17_106X_mc2017_realistic_v9-v1/PREMIX" : {}
+          "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL17_106X_mc2017_realistic_v9-v1/PREMIX" :
+          {
+                  "SecondaryLocation": [
+                     "T1_DE_KIT_Disk",
+                     "T1_US_FNAL_Disk"
+                      ]
+          },
+          "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL17CP5_106X_mc2017_realistic_v9-v2/PREMIX" :
```

Comment:
I'd choose FNAL again for US site. I have no clue how Nebraska will perform. Let's stay safe, IMO.

Reply:
We stayed safe last time and I wanted to test it with one WF but if you still are worried sure FNAL.

Reply:
yes, I think we should use FNAL

Reply:
@scarletnorberg  can you please Nebraska w/ FNAL for this campaign as well?

Reply:
Yes this and the FR one are changed in both FS campaigns instead of only one of the FS campaigns.